core/server/data/migrations/versions/4.33/01-add-type-column-to-products.js

```diff
@@ -0,0 +1,13 @@
+const utils = require('../../utils');
+
+module.exports = utils.createAddColumnMigration(
+    'products',
+    'type',
+    {
+        type: 'string',
+        maxlength: 50,
+        nullable: false,
+        defaultTo: 'paid',
+        validations: {isIn: [['paid', 'free']]}
```

Comment:
This isn't necessary as it's not used to construct the column

Reply:
Updated, thanks!